pkg/distro/distro_test.go

```diff
@@ -171,6 +172,15 @@ func TestImageTypePipelineNames(t *testing.T) {
 					// sure they match.
 					assert.Equal(imageType.Exports()[0], pm.Pipelines[len(pm.Pipelines)-1].Name)
 
+					// The pipelines named in allPipelines must exist in the manifest, and in the
+					// order specified (eg. 'build' first) but it does not need to be an exact
+					// match. Only the pipelines with rpm or ostree metadata are required.
```

Comment:
I think technically the order doesn't matter, but might as well check for it.

Thanks for this!

Reply:
Order was inherent in the previous behavior, but I also swear I saw a comment about it someplace. Which I cannot find now :)